slatedb/src/compactor_executor.rs

```diff
@@ -187,13 +187,11 @@ impl TokioCompactionExecutorInner {
         let mut current_writer = self
             .table_store
             .table_writer(SsTableId::Compacted(self.rand.rng().gen_ulid()));
-        let mut current_size = 0usize;
+
+        let mut compacted_block_size = 0usize;
```

Comment:
Nit: I find `compacted_block_size` a bit confusing. I first read it as the size of a single block. Maybe just `bytes_written`?

Reply:
changed.